pkg/fanal/analyzer/language/golang/mod/mod_test.go

```diff
@@ -101,14 +105,40 @@ func Test_gomodAnalyzer_Analyze(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:    "no `pkg` folder",
```

Comment:
This test passes regardless of your change as no such dir error is already warning, right? We should add a test which doesn't pass before your change and passes after your change.

Reply:
Done